solarwinds_apm/distro.py

```diff
@@ -65,10 +71,42 @@ def _log_runtime(self):
         self._log_python_runtime()
         logger.info("OpenTelemetry %s/%s", sdk_version, inst_version)
 
+    def _get_token_from_service_key(self):
+        """Return token portion of service_key if set, else None"""
+        service_key = environ.get("SW_APM_SERVICE_KEY")
```

Comment:
This is partially duplicated from ApmConfig because Distro._configure lifetime ends before Configurator and ApmConfig are called during auto-instrumentation.

Reply:
Are configuration and instrumentation two separate steps ?